docker-compose.yml

```diff
@@ -3,25 +3,19 @@ version: '3'
 services:
   postgres:
     environment:
-      POSTGRES_DB: metadeploy
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: sample_db_password
     volumes:
       - ./postgres:/var/lib/postgresql/data:delegated
     image: postgres:12.9
-    ports:
-      - '5432:5432/tcp'
+    restart: always
```

Comment:
With `restart: always` the postgres servers will always run, even if we're currently not working on the project. Is that what we want?

Reply:
That's what we've been using on Metecho, OddBooks, and GSM. 🤷 

Without this, I was often getting an error on initial startup: `FATAL:  data directory "/var/lib/postgresql/data" has wrong ownership`

With this change, I sometimes see that error initially in the postgres container logs, but then it restarts and works.

Reply:
Sounds like an underlying ownership issue since we run as `root` inside containers. I don't know how to fix that though, so this should be fine for now 👍